### PR TITLE
Added new changes to navbar, sections and scrollbar

### PR DESCRIPTION
**Pull request by m-Thelinn**
About section completed
Added white color to navbar
Added new MT logo to navbar
Added new scrollbar style
Some translations in various sections

### CODE REVIEW

**Review by sergioconv — APPROVED** [review]

File: assets/css/style.css [hunk 1]
```diff
@@ -665,4 +686,56 @@ section {
...
+::-webkit-scrollbar-thumb {
+    background-color: var(--main-color);
+    width: 1rem;
+    border-radius: 5px;
+    border: 3px solid #fff;
+}
+
+
+/*
+#scrollPath {
```
Comment (MCheca) [inline assets/css/style.css:704]: Si no se usa bórralo

File: assets/js/script.js [hunk 2]
```diff
@@ -42,47 +42,54 @@ $(document).ready(function() {
...
+
+    $("#menu-item a").click(function() {
+        $('.menu').toggleClass('c m');
+        $('.menu span').toggleClass('ion-navicon ion-android-close');
+        $('#menu-item').toggleClass("show-menu hide-menu");
+    });
+
+
+});
+//SCROLLBAR
```
Comment (MCheca) [inline assets/js/script.js:89]: Si no se usa bórralo

File: src/index.html [hunk 3]
```diff
@@ -57,12 +58,15 @@
     <div class="main">
         <header class="bg-img header">
             <nav class="navbar navbar-default navbar-vira">
+                <!--<div id="progressbar"></div>
```
Comment (MCheca) [inline src/index.html:61]: No dejes cosas comentadas que no se usan

**Review by MCheca — CHANGES_REQUESTED** [review]
Revisa los comentarios

**Review by MCheca — APPROVED** [review]
All is ok!!

**Pull request merged**